### ISSUE TITLE
Newline pasting broken in kitty+neovim

### ISSUE THREAD

**Issue by leoshatrushin** [tmux/tmux#4163]
### Issue description

Hello, I have
```
set -s extended-keys on
set -as terminal-features 'xterm*:extkeys'
set -gq allow-passthrough on
```
And as of this update pasting a newline in kitty+tmux+neovim puts "^[[27;5;106~". If I add
```
set extended-keys-format csi-u
```
It puts "^[[106;5u". It works fine in vim. Does not work in no-config neovim.

### Required information

Please provide the following information:

* tmux version (`tmux -V`): 3.5
* Platform (`uname -sp`): Darwin arm
* $TERM inside and outside of tmux (`echo $TERM`): inside=xterm-kitty, outside=tmux-256color
* Logs from tmux (`tmux kill-server; tmux -vv new`).

**Comment by nicm** [comment]
Does this still happen in master?

**Comment by leoshatrushin** [comment]
Yes

**Comment by nicm** [comment]
Show logs please.

**Comment by leoshatrushin** [comment]
My bad, I probably didn't kill the non-master server before running master tmux, though I thought I did. It does seem to work fine on master.

**Comment by nicm** [comment]
OK great, someone else did report this but unfortunately I forgot about the fix for 3.5a. It is probably too big a change anyway...

You should also be able to work around this by turning off `extended-keys`.
